oci/tempo/documentation.yaml

```diff
@@ -0,0 +1,54 @@
+version: 1
+# --- OVERVIEW INFORMATION ---
+application: tempo
+description: >
+  Tempo is a distributed tracing backend by Grafana, supporting Jaeger,
+  Zipkin, and OpenTelemetry protocols.
+  Read more on the [official website](https://grafana.com/oss/tempo/).
+# --- USAGE INFORMATION ---
+docker:
+  parameters:
+    - -p 3200:3200
+    - -p 14268:14268
+    - -p 9095:9095
+    - -p 4317:4317
+    - -p 4318:4318
+    - -p 9411:9411
+  access: Access your Tempo API at `http://localhost:3200`. Ingestion endpoints for each format open a separate port.
+parameters:
+  - type: -e
+    value: 'TZ=UTC'
+    description: Timezone.
+  - type: -p
+    value: '3200:3200'
+    description: Tempo HTTP API port.
+  - type: -p
+    value: '9095:9095'
+    description: Tempo GRPC API port.
+  - type: -p
+    value: '14268:14268'
+    description: Jaeger thrift format ingestion port.
+  - type: -p
+    value: '4317:4317'
+    description: OpenTelemetry GRPC ingestion port.
+  - type: -p
+    value: '4318:4318'
+    description: OpenTelemetry HTTP ingestion port.
+  - type: -p
+    value: '9411:9411'
+    description: Zipkin format ingestion port.
+debug:
+  text: |
+    ### Debugging
+    
+    To debug the container:
+
+    ```bash
+    docker exec -it tempo-container /bin/pebble logs -f tempo
```

Comment:
Thanks for the update. Why do you need `/bin` though? That should be in $PATH. Just `pebble logs ...` should work, doesn't it?

Reply:
True, it does and I updated the description. Thanks, good catch!